### DISCUSSION TITLE
[Answered] Does Activity Type Influence What Data Gets Recorded?

### DISCUSSION THREAD

**Discussion by vkbellis** [OpenTracksApp/OpenTracks#870] [Q&A] [answered]
![Roof-mounted_devices](https://user-images.githubusercontent.com/82275968/124383957-3b6e6980-dc9d-11eb-8a59-454de6bbea7f.jpg)

I'm designing some geoid model tests that will place Android devices mounted to the roof of a vehicle. Yesterday, the first of these tests were run, but I had forgotten to change the Activity Type from walking to driving. This morning I wonder, does Activity Type influence what data get recorded?

![Activity types](https://user-images.githubusercontent.com/82275968/124384060-ae77e000-dc9d-11eb-8290-4021b08eeaa3.PNG)

**Comment by dennisguse** [comment, marked as answer]
The activity type (aka category) does not affect anything. It is just for the user.

**Reply by vkbellis** [reply]
Thanks Dennis!

**Reply by dennisguse** [reply]
PS: Nice construction by the way :)

**Reply by vkbellis** [reply]
Nerdvana :)
